source/supported-hardware/whl-cfl-crb.rst

```diff
@@ -0,0 +1,121 @@
+.. _whl-cfl-crb:
+
+WHL-CFL CRB Boards
+-----------------------
+
+.. note:: 8th and 9th Generation Intel® Core Processor Families formally known as |WHL| and |CFLR| platform.
+
+Supported Boards
+^^^^^^^^^^^^^^^^^^^^^
+
+|SPN| supports **Whiskey Lake, Coffee Lake Refresh Desktop S8+2 and Coffee Lake Refresh Mobile H6+2** CRB platforms.
+  
+
+Building
+^^^^^^^^^^
+
+To build |SPN| for |WHL| and |CFLR| platforms::
+
+    python BuildLoader.py build cfl
+
+The output images are generated under ``Outputs`` directory.
+
+
+Stitching
+^^^^^^^^^^
+
+1. Gather |WHL| or |CFLR| IFWI firmware image
+
+  This image contains additional firmware ingredients that are required to boot on |WHL| or |CFLR|.
+
+.. note::
+  ``StitchLoader.py`` currently only supports stitching with boot guard feature **disabled**. 
+
+
+2. Stitch |SPN| images into downloaded BIOS image::
+
+    python Platform/CoffeelakeBoardPkg/Script/StitchLoader.py -i <BIOS_IMAGE_NAME> -s Outputs/cfl/Stitch_Components.zip -o <SBL_IFWI_IMAGE_NAME>
+
+    where -i = Input file, -o = Output file.
+
+For example, stitching |SPN| IFWI image ``sbl_cfl_ifwi.bin`` from |CFLR| firmware image::
+
+    python Platform/CoffeelakeBoardPkg/Script/StitchLoader.py -i xxxx.bin -s Outputs/cfl/Stitch_Components.zip -o sbl_cfl_ifwi.bin
+
+For more details on stitch tool, see :ref:`stitch-tool` on how to stitch the IFWI image with |SPN|.
+
+
+Flashing
+^^^^^^^^^
+
+Flash the generated ``sbl_cfl_ifwi.bin`` to the target board using DediProg SF100 or SF600 programmer.
+
+.. note:: Please check the alignment/polarity when connecting Dediprog to the board. Please power off the board before connecting the Dediprog.
+
+SPI Header to connect dediprog for CRB platforms is provided in the table below.
+
+  +---------------------------+--------------+
+  |            Board          |   Connector  |
+  +---------------------------+--------------+
+  |            |WHL|          |    **J3H1**  |
+  +---------------------------+--------------+
+  |           |CFLSR|         |    **J7H1**  |
+  +---------------------------+--------------+
+  |           |CFLHR|         |    **J7H2**  |
+  +---------------------------+--------------+
+
+.. note:: Please disconnect Deidprog before powering up the board again.
+
+
+Debug UART
+^^^^^^^^^^^
+
+Serial port connector for CRB platforms is provided in the table below
```

Comment:
Add one line to mention the UART settings , like 115200N8

Reply:
Done